src/pages/careers/javascript-engineer.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { createUseStyles } from 'react-jss';
+import { renderToString } from 'react-dom/server';
+
+import { SEO, SitewideFooter, LayoutControl, SitewideHeader } from 'components';
+import Hero from 'components/careers/Hero';
+import Footer from 'components/careers/Footer';
+import Main from 'components/careers/Main';
+import { backstageLink } from 'components/careers/links';
+
+const useStyles = createUseStyles(() => ({
+  sitewideHeaderWrapper: {
+    marginBottom: 40,
+    paddingLeft: 16,
+    paddingRight: 16,
+  },
+}));
+
+const ROLE_NAME = 'JavaScript & TypeScript Engineer';
+const TYPEFORM_SLUG = 'javascript-engineer';
+const HEADLINE = 'Get paid to become a top open-source contributor';
+
+const ROLE = (() => {
+  return [
+    'Founding engineer at an early stage, VC-backed startup.',
+    `Become a top contributor to the ${backstageLink} that Roadie is built on.`,
+    `You'll build new product features from start to finish: through conception, 
+     research, maintenance, operation, and polish.`,
+    `Learn rapidly and work in all parts of the stack.`,
+    `Work closely with experienced engineers who previously worked at Workday and Spotify.`,
```

Comment:
did you want to mention weaveworks?

Reply:
Not really. It's the same engineer who worked at Weaveworks and Spotify so 😄  I realise I did it in the other file.